bluebrain/repo-patches/packages/neuron/package.py

```diff
@@ -89,15 +99,42 @@ class Neuron(CMakePackage):
     # Numpy is required for Vector.as_numpy()
     depends_on("py-numpy",    when="+python", type=("build", "run"))
     depends_on("py-cython",   when="+rx3d", type="build")
-    depends_on("tau",         when="+profile")
-    depends_on("coreneuron+legacy-unit~caliper", when="+coreneuron+legacy-unit~caliper")
-    depends_on("coreneuron~legacy-unit~caliper", when="+coreneuron~legacy-unit~caliper")
-    depends_on("coreneuron+legacy-unit+caliper", when="+coreneuron+legacy-unit+caliper")
-    depends_on("coreneuron~legacy-unit+caliper", when="+coreneuron~legacy-unit+caliper")
```

Comment:
Does this mean that for current `neuron` version there is no dependency to `coreneuron`? I don't see any `depend_on('coreneuron', ...)` in the changes

Reply:
yup..this was an error! 👍